Lib/collections/__init__.py

```diff
@@ -691,6 +691,42 @@ def __delitem__(self, elem):
         if elem in self:
             super().__delitem__(elem)
 
+    def __eq__(self, other):
+        'True if all counts agree. Missing counts are treated as zero.'
+        if not isinstance(other, Counter):
+            return NotImplemented
+        return all(self[e] == other[e] for c in (self, other) for e in c)
```

Comment:
Just curious.  Wouldn't this (and similarly below) be clearer as

```python
    return all(self[e] == other[e] for e in chain(self, other))
```

?

Reply:
I wanted to avoid the dependency and for this to run well on PyPy.